Versioning.md

```diff
@@ -4,40 +4,35 @@ This document describes the versioning policy for this repository.
 
 ## Goals
 
-### API Stability
+### API and SDK Compatibility
 
 Once the API for a given signal (spans, logs, metrics, baggage) has been
 officially released, that API module will function with any SDK that has the
 same MAJOR version and equal or greater MINOR or PATCH version. For example,
 application compiled with API v1.1 is compatible with SDK v1.1.2, v1.2.0, etc.
 
 For example, libraries that are instrumented with `opentelemetry 1.0.1` will
-function in applications using `opentelemetry 1.11.33` or `opentelemetry 1.3.4`.
-
-### SDK Stability
-
-Public portions of the SDK (constructors, configuration, end-user interfaces)
-must remain backwards compatible. Internal types are allowed to break.
+function in applications using `opentelemetry 1.11.33` or `opentelemetry 1.3.4`,
+buy may not work in applications using `opentelemetry 2.0.0`.
 
 ### ABI Stability
 
 Refer to the [ABI Policy](./docs/abi-policy.md) for more details. To summarise:
 
-* ABI stability is guaranteed for the API.
-* ABI stability is not guaranteed for the SDK. In case of ABI breaking changes,
-  instead of bumping up the major version, a new `inline namespace` version will
-  be created, and both old API and new API would be made available
-  simultaneously.
+* The API is header only, and uses ABI compliant interfaces. However, ABI stability
+  is not guaranteed for SDK.
+* In case of ABI breaking changes, a new `inline namespace` version will
+  be introduced, and the existing linked applications can continue using the older version
```

Comment:
Could we give an example here on `inline namespace`?

Reply:
Sorry seems i missed this comment. ABI versioning using inline namespace is handled here in the code: https://github.com/open-telemetry/opentelemetry-cpp/blob/5767f8e4a2cd32e8ab8647fc5eb672a5e3c21ed6/api/include/opentelemetry/version.h#L13-L14. Should I be adding reference to this ?

Reply:
Yes, I think we could add a reference in the next doc update.